### PR TITLE
🐛Check for window null before creating tracking pixel

### LINKED ISSUE #23935: Error: null is not an object (evaluating 'new a.Image')
```
Error: null is not an object (evaluating 'new a.Image')
at Image (https://raw.githubusercontent.com/ampproject/amphtml/1908121824440/src/window-interface.js:95)
at iframe (https://raw.githubusercontent.com/ampproject/amphtml/1908121824440/src/pixel.js:61)
at win (https://raw.githubusercontent.com/ampproject/amphtml/1908121824440/extensions/amp-analytics/0.1/transport.js:246)
at (https://raw.githubusercontent.com/ampproject/amphtml/1908121824440/extensions/amp-analytics/0.1/transport.js:132)
at requestUrl (https://raw.githubusercontent.com/ampproject/amphtml/1908121824440/extensions/amp-analytics/0.1/requests.js:260)
```

My guess is the `window` is destroyed by the time `Transport.sendRequest` is called. A null check may fix it.

/cc @ampproject/wg-analytics

### PR DESCRIPTION
**Pull request by zhouyx**
Closes #23935

### CODE REVIEW

File: src/pixel.js [hunk 1]
```diff
@@ -29,6 +29,7 @@ const TAG = 'pixel';
  * @return {!Element}
  */
 export function createPixel(win, src, referrerPolicy) {
+  // Caller need to verify window is not destoried when creating pixel
```
Comment (Enriqe) [inline src/pixel.js]: nit: destoried/destroyed

Comment (dvoytenko) [reply src/pixel.js]: Don't let STAMP folk see this :)

Comment (Enriqe) [reply src/pixel.js]: LOL

Comment (zhouyx) [reply src/pixel.js]: LOL!

**Comment by choumx** [comment]
Thanks Yuxuan!

**Pull request merged**